x-pack/plugins/security_solution/public/overview/components/detection_response/alerts_by_status/formatted_count.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FormattedNumber } from '@kbn/i18n-react';
+import React from 'react';
+
+const FormattedCountComponent: React.FC<{ count: number | null }> = ({ count }) => {
```

Comment:
What happens if we have numbers in the order of `M`,`B` or `T`?
I have found that: `x-pack/plugins/security_solution/public/resolver/view/submenu.tsx:27` maybe we could find a more "common" place for it

Reply:
we could also use `notation="compact"` feature of _Intl.NumberFormat_ but I am not sure all supported browsers are implementing it yet

Reply:
yes exactly, thanks!